extensions/elevenlabs_tts/script.py

```diff
@@ -55,17 +60,20 @@ def remove_surrounded_chars(string):
     return re.sub('\*[^\*]*?(\*|$)', '', string)
 
 
-def state_modifier(state):
```

Comment:
This is now necessary, and the lines that change shared.args.no_stream have to be removed